### DISCUSSION TITLE
T-Pot Help - Log Files & Honeypots

### DISCUSSION THREAD

**Discussion by TS-02** [telekom-security/tpotce#1045] [Q&A]
Hi.

This is my first time diving into honeypots and I was hoping I could get some advice and help regarding T-Pot and its log files. 

**>** Is T-Pot classified as a honeypot itself or a collection of honeypots?
**>** Where can I find the log files for T-Pot? 
**>** Would there be a difference in me for example, having a separate Cowrie honeypot or using Cowrie built in to T-Pot itself?

If anyone could answer my queries, I would really appreciate it.

Thanks.

**Comment by t3chn0m4g3** [comment]
> Is T-Pot classified as a honeypot itself or a collection of honeypots?

Collection
> Where can I find the log files for T-Pot?

`/data`
> Would there be a difference in me for example, having a separate Cowrie honeypot or using Cowrie built in to T-Pot itself?

No, even if you wanted to have your own config (see Wiki for volumes) T-Pot use the same Cowrie sources from Github.

**Comment by TS-02** [comment]
Hi @t3chn0m4g3, 

Thank you for your reply.

I have some additional questions.

**>** When the honeypot was set up, the Cockpit section of the web portal was able to load. However, now, I can't seem to connect to it. What would you recommend I do?

**>** What is the best way to view the logs of one of the honeypots, for example **Cowrie**?

I look forward to hearing from you.

**Reply by t3chn0m4g3** [reply]
> When the honeypot was set up, the Cockpit section of the web portal was able to load. However, now, I can't seem to connect to it. What would you recommend I do?

You probably were locked out by fail2ban, try to clean blocks with `fail2ban-client`.

> What is the best way to view the logs of one of the honeypots, for example Cowrie?

Kibana

**Comment by TS-02** [comment]
Hi @t3chn0m4g3,

Thank you for your reply.

**>** If you don't mind me asking, how do I go about cleaning the blocks? with **fail2ban-client**?

**>** With regards to viewing the logs for **cowrie**, could you offer some advice on the best way to do this via the terminal?

I look forward to hearing from you.
